DFC.Digital/DFC.Digital.Core/Logging/DFCLogger.cs

```diff
@@ -39,6 +39,13 @@ public void Info(string message)
             logService.Info(message);
         }
 
+        public string LogExceptionWithActivityId(Exception ex)
+        {
+            var activityId = Guid.NewGuid().ToString();
+            logService.LogException(NLog.LogLevel.Error, $"Service status check failed for activity id - {activityId}", ex);
```

Comment:
This method can be re-used, so either change the method signature as StatusCheckException(ex) or accept a string message., and append the activity id to the beginning of the message.